Variable set but not used in OBForceField::DistanceGeometry()

I've been working on eliminating warnings from "-W -Wall" (for "make openbabel") with the aim of adding this to Travis. Here's one I don't immediately see the correct solution for (does it indicate a bug, for example?), but @ghutchis might:
```
/home/noel/Tools/openbabel-git/src/forcefield.cpp: In member function 'void OpenBabel::OBForceField::DistanceGeometry()':
/home/noel/Tools/openbabel-git/src/forcefield.cpp:1962:26: error: variable 'l_bc' set but not used [-Werror=unused-but-set-variable]
             double l_ab, l_bc, l_ac; // lower limits
```

--- a/src/distgeom.cpp
+++ b/src/distgeom.cpp
@@ -783,11 +783,11 @@ namespace OpenBabel {
           }
 
           // Triangle rule: length can't be shorter than the difference between the legs
-          if (l_bc < (l_ab - u_ac)) {
-            l_bc = l_ab - u_ac;
+          if (l_bc < (l_ab - l_ac)) {
+            l_bc = l_ab - l_ac;
             _d->SetLowerBounds(b, c, l_bc);
-          } else if (l_bc < (l_ac - u_ab)) {
-            l_bc = l_ac - u_ab;
+          } else if (l_bc < (l_ac - l_ab)) {
+            l_bc = l_ac - l_ab;
             _d->SetLowerBounds(b, c, l_bc);
           }
 

--- a/src/forcefield.cpp
+++ b/src/forcefield.cpp
@@ -1991,8 +1991,8 @@ namespace OpenBabel
               self_consistent = false;
             }
 
-            if (l_ac < (l_ab - u_bc)) {// l_ac >= l_ab - u_bc
-              l_ac = l_ab - u_bc;
+            if (l_ac < (l_ab - l_bc)) {// l_ac >= l_ab - l_bc
+              l_ac = l_ab - l_bc;
       	      self_consistent = false;
             }
 

